src/request.ts

```diff
@@ -73,6 +76,11 @@ const attachErrorDetails = (err: unknown) => {
   throw err
 }
 
+export const addVersionToUrl = (url: string, version: Version = "v3"): string => {
```

Comment:
Is it right to default to v3 rather than v2?

Reply:
This is something the API team decided upon. clients can still use v2 handily without upgrading this library, but then will be using v3 for the same effect anyway if they do upgrade, and if they want old functionality they will need to specify